security-policies/bundle/compliance/cis_azure/rules/cis_3_14/rule.rego

```diff
@@ -0,0 +1,24 @@
+package compliance.cis_azure.rules.cis_3_14
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import data.compliance.policy.azure.storage_account.ensure_service_log as audit
+import future.keywords.if
+
+finding = result if {
+	# filter
+	data_adapter.is_storage_account
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(logs_are_enabled),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+default logs_are_enabled = false
+
+logs_are_enabled if {
```

Comment:
The benchmark mentions on Audit From Azure Portal step 4: `Ensure the Status is set to On, if set to Off.` I don't see it being covered here. Is it not needed? 🤔

Reply:
CIS rule 3.5 mentions the classic interface, while 3.13 and 3.14 mention the new (?) non-classic interface. The JSON object does not contain a root status field beside the interface. It is like this:

```json
{
    "id": "/subscriptions/...",
    "name": "name_name",
    "properties": {
        "logs": [
            {
                "category": "StorageRead",
                "categoryGroup": null,
                "enabled": true,
                "retentionPolicy": {
                    "days": 0,
                    "enabled": false
                }
            },
            {
                "category": "StorageWrite",
                "categoryGroup": null,
                "enabled": true,
                "retentionPolicy": {
                    "days": 0,
                    "enabled": false
                }
            },
            {
                "category": "StorageDelete",
                "categoryGroup": null,
                "enabled": true,
                "retentionPolicy": {
                    "days": 0,
                    "enabled": false
                }
            }
        ],
        "metrics": [
            {
                "timeGrain": null,
                "enabled": false,
                "retentionPolicy": {
                    "days": 0,
                    "enabled": false
                },
                "category": "Transaction"
            }
        ],
        "workspaceId": "/subscriptions/...",
        "logAnalyticsDestinationType": null
    }
}
```

So, I believe this `status` mentioned in the classic UI is a conclusive element (based on the presence or absence of diagnostic settings per service) and not a standalone field.